cea/interfaces/dashboard/map_layers/base.py

```
@@ -45,39 +48,98 @@ def expected_parameters(self) -> dict:
 
     @property
     @abc.abstractmethod
-    def input_files(self) -> Collection[Tuple[locator_func, locator_func_args]]:
+    def input_file_locators(self) -> Collection[Tuple[locator_func, locator_func_args]]:
         """Returns a collection of callables that are used to generate the input file paths used for this layer"""
 
     def _validate_parameters(self) -> None:
         """Validates the parameters"""
```

Ruff (0.7.0)

`MapLayer._validate_parameters` is an empty method in an abstract base class, but has no abstract decorator

(B027)